core/identity/identity.go

```diff
@@ -373,17 +382,17 @@ func (s *service) observeIdentitiesLoop() {
 		case <-s.closing:
 			return
 		case <-s.identityForceUpdate:
-			observe()
+			observe(true)
```

Comment:
it will be called on each RegisterIdentity call

Reply:
Yes, it could be a problem?